libs/dh/shared/data-access-api/src/lib/generated/v1/model/reading-occurrence.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * DataHub BFF
+ * Backend-for-frontend for DataHub
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+export type ReadingOccurrence = 'P1Y' | 'P1M' | 'PT1H' | 'PT15M';
+
+export const ReadingOccurrence = {
+    P1Y: 'P1Y' as ReadingOccurrence,
+    P1M: 'P1M' as ReadingOccurrence,
+    Pt1H: 'PT1H' as ReadingOccurrence,
+    Pt15M: 'PT15M' as ReadingOccurrence
```

Comment:
question: Everything looks fine. Only one non-blocking thing. Do you think this can be configured to be all capital letters?

Reply:
We'll adjust the configuration if it becomes an issue.